lib/between_meals/knife.rb

```diff
@@ -95,13 +99,16 @@ def cookbook_upload(cookbooks)
     def berks_cookbook_upload(cookbooks)
       # cookbooks: array
       # cookbook_paths: array
+      if @berks_config
+        @berks_config = '--config=' + @berks_config
```

Comment:
you probably need this to be a local variable, now that I think about this. If anyone makes a knife object and calls berks_cookbook_upload more than once, this will break.